ZooProject/src/zoo/Zoo.java

```diff
@@ -296,141 +382,60 @@ public void setBesucher(Besucher[] besucher) {
-				for (int i = 0; i < this.gehege.length; i++) {
-					if (i == this.gehege.length - 1 && this.gehege[i] != null) {
-						System.out.println(this.getName() + " kann keine weiteren Gehege anlegen ");
-
-					} else if (this.gehege[i] == null) {
-						this.gehege[i] = landsaeugetiergehege;
-						break;
-					}
-				}
-				System.out.println(this.getName() + " hat ein neues LandsaeugeTierGehege mit der Bezeichnung " + name[k]);
-
-			} else if (Gehege_typ == 3) {
-				Gehege aquarium = new Aquarium(name[k], zoo);
-				for (int i = 0; i < this.gehege.length; i++) {
-					if (i == this.gehege.length - 1 && this.gehege[i] != null) {
-						System.out.println(this.getName() + " kann keine weiteren Gehege anlegen ");
-
-					} else if (this.gehege[i] == null) {
-						this.gehege[i] = aquarium;
-						break;
-					}
-				}
-				System.out.println(this.getName() + " hat ein neues Aquarium mit der Bezeichnung " + name[k]);
-			}
-		}
-	}
-
 	public void outputZustand() {
-		// Platzhalter
+		
+		//Platzhalter
 		System.out.println();
-
-		// Zustand des Vogelgeheges
-		for (int p = 0; p < gehege.length; p++) {
-			if (gehege[p] != null) {
-
-				System.out.print(this.name + " hat das Gehege " + this.gehege[p].getName() + " ");
-
-				// Jedes Tier im Vogelgehege wird durchgegangen ... allerdings wird zuerst
-				// geschaut
-				// ob da �berhaupt Tiere drinnen sind
-
-				for (int k = 0; k < gehege[p].getTiere().length; k++) {
-					if (gehege[p].getTiere()[k] != null) {
-						System.out.print(gehege[p].getTiere()[k].getName() + ", ");
+		
+		
+	//Zustand des Vogelgeheges
+		for(int p = 0; p< gehege.length; p++) {
+			if(gehege[p] != null) {
+				
+			System.out.print(this.name+" hat das Gehege "+ this.gehege[p].getName()+" ");
+			
+			//Jedes Tier im Vogelgehege wird durchgegangen ... allerdings wird zuerst geschaut
+			//ob da überhaupt Tiere drinnen sind
+			
+				for(int k = 0; k<gehege[p].getTiere().length; k++) {
+					if(gehege[p].getTiere()[k] != null) {
+					System.out.print( gehege[p].getTiere()[k].getName()+", ");
 					}
 				}
 				System.out.println(" ");
-			}
+			}	
 		}
-
-		// angestellte Zustand und wo sich welches Personal wo befindet
-
-		for (int arb = 0; arb < angestellte.length; arb++) {
-			if (angestellte[arb] != null) {
-				System.out.println(this.name + " hat den Angestellten " + angestellte[arb].getName()
-						+ " ,welcher sich um folgende Gehegen k�mmert: ");
-
-				for (int g = 0; g < angestellte[arb].getGehege().length; g++) {
-					if (angestellte[arb].getGehege()[g] != null) {
-						System.out.print(angestellte[arb].getGehege()[g].getName() + ", ");
+		
+	//	angestellte Zustand und wo sich welches Personal wo befindet
+		
+		for(int arb = 0; arb < angestellte.length; arb++) {
+			if(angestellte[arb] != null) {	
+				System.out.println(this.name+" hat den Angestellten "+ angestellte[arb].getName()+" ,welcher sich um folgende Gehegen kümmert: ");
```

Comment:
Personal kümmert sich nicht um Gehege sondern um Tiere dswg hab ich auch diesen int bei Tier eingeführt, da Betreuer A sich bspw nur um Vögel und Adler kümmert und es theorisch ja sein könnt, dass ein Tier in 2 Gehegetypen leben kann.